google/gax/__init__.py

```diff
@@ -150,8 +163,9 @@ class RetryOptions(
     """Per-call configurable settings for retrying upon transient failure.
 
     Attributes:
-      retry_codes: a list of exceptions upon which a retry should be attempted.
-      backoff_settings: a BackoffSettings object configuring the retry
+      retry_codes (list[string]): a list of exception codes upon which a retry
```

Comment:
In that case, should we refer to them as "Google API canonical error codes" as in that proto file?